.github/workflows/pr-review-client-delete-unused.yml

```diff
@@ -0,0 +1,45 @@
+name: Delete unused PR review environments
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 4 * * *" # Nightly at 4am
+
+env:
+  AWS_DEFAULT_REGION: ca-central-1
+  DELETE_AFTER_DAYS: 21
+  FUNCTION_PREFIX: "forms-client-pr"
+  IMAGE: forms/client
+
+jobs:
+  delete-unused-test-client:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS credentials
+        id: aws-creds
+        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838 # v1.7.0
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ca-central-1
```

Comment:
For this, we could use the new OIDC roles that exist in the account.  The action setup looks like this:
https://github.com/cds-snc/url-shortener/blob/7fb500e8fb4971852c9fd1fb95010a0c7c928a04/.github/workflows/tf_apply_staging.yml#L45-L50

And the ARN of the role would be:
```
arn:aws:iam::687401027353:role/platform-forms-client-apply
```